scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -84,6 +84,19 @@ $regKeyNuGet32 = "HKLM:SOFTWARE\Wow6432Node\Microsoft\StrongName\Verification\*,
 
 
 $has32bitNode = Test-Path "HKLM:SOFTWARE\Wow6432Node"
+
+# update submodule NuGet.Build.Localization
+$NuGetClientRoot = (Get-Location).Path
+$Submodules = Join-Path $NuGetClientRoot submodules -Resolve
+
+$NuGetLocalization = Join-Path $Submodules NuGet.Build.Localization -Resolve
+$updateOpts = 'pull', 'origin', 'master'
+           
+Set-Location $NuGetLocalization
```

Comment:
use the -C switch of git instead. see: https://git-scm.com/docs/git 

Run as if git was started in <path> instead of the current working directory. When multiple -C options are given, each subsequent non-absolute -C <path> is interpreted relative to the preceding -C <path>.